platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -877,6 +877,7 @@
             $scope.advancedSettings = Entities.getAdvanced({id: $scope.selectedEntity.id},
                 function () {
                     $scope.blockLookups = false;
+                    $scope.notDisabled = true;
```

Comment:
1. this is not the best name, in boolean fields you shouldn't use 'not', I need few seconds to understand what true means for this field
2. apart from 1., the name is not adequate to purpose of this field.

plus i think there should be two boolean fields for add new lookup button and for add new lookup field button. Because now when for example 'new lookup field' button is disabled, the button 'new lookup' is also disabled.